lib/travis/cli/repo_command.rb

```diff
@@ -83,6 +82,14 @@ def detect_slug
           end
         end
 
+        def parse_remote(url)
+          URI.parse(url).path
+        rescue URI::InvalidURIError
```

Comment:
This has been updated to explicitly handle `git@github.com` remotes instead of relying on the `InvalidURIError`.